ui/src/views/devices/ListDevices.tsx

```diff
@@ -69,18 +58,17 @@ class ListDevices extends Component<IProps, IState> {
           if (record.lastSeenAt !== undefined) {
             let ts = new Date(0);
             ts.setUTCSeconds(record.lastSeenAt.seconds);
-            return moment(ts).format("YYYY-MM-DD HH:mm:ss");
```

Comment:
I moved away from `fromNow()` on purpose. In some cases this shows something like 'in a few seconds' where it should have been 'a few seconds ago'. This happens when the user machine is not time-synced or when there is a minor time difference between the server and user machine.